client-e2e/test/specs/receive.ts

```diff
@@ -33,7 +33,7 @@ describe("Receive flow", () => {
         await (await Page.receiveButton()).click();
         const input = await Page.receiveCodeInput();
         await input.click();
-        await browser.keys(["7-guitarist-revenge"]);
+        await browser.keys(["100-guitarist-revenge"]);
```

Comment:
why 100 instead of 7?

Reply:
Thought, if run against real mailbox service (w.*, w3.*), theoretically might interrupt sender and test result might be different. Higher number, less probability.

Reply:
Then use a nameplate that can't be automatically allocated (and a comment to this effect would be nice).

Reply:
Maximum nameplate number is 999? 1000 should be good

Reply:
yes